CrawlerTests/AnalyzersTests/PunctuationAnalyzerTests.cs

```diff
@@ -7,33 +7,58 @@
 using Moq;
 using System.Collections.Generic;
 using System.Linq;
+using Crawler.PartOfSpeechTagger;
 using Xunit;
 
 namespace CrawlerTests.AnalyzersTests
 {
     public class PunctuationAnalyzerTests
     {
-        private readonly ILexer lexer;
-        private readonly LexerConfig config;
-        private readonly PunctuationAnalyzer punctuationsAnalyzer = new PunctuationAnalyzer();
+	    private const string PASSIVE_VOICE_SENTENCE = "The house will be cleaned by me every Saturday.";
+	    private const string ACTIVE_VOICE_SENTENCE = " Sue changed the flat tire. ";
+
+	    private ILexer lexer;
```

Comment:
Why use lexer? we talked about it before it's not good for the unit tests

Reply:
Because it efficient, the code testing about 20 sentences and write them as a Tokens list will be unnecessary work. You're right that we have talked about it, but I still think it's not should have to be a problem, because it's already been checked in its own tests.

Reply:
Still not sure about this but at this point I don't really care